lib/src/Datepicker/Datepicker.tsx

```diff
@@ -107,7 +103,8 @@ export const Datepicker: React.FunctionComponent<DatepickerProps> = ({
 
     const handleChangeCustomDay = (e: React.ChangeEvent<HTMLInputElement>): void => {
         if (!monthPicker) {
-            setCustomDay(Number(e.target.value));
+            const v: number = e.target?.value && !Number.isNaN(e.target?.value) ? Number(e.target.value) : null;
```

Comment:
i think you should check if e.target.value is a number before using it inside Number.isNan(e.traget.value)
```suggestion
            const v: number = e.target?.value && !Number.isNaN(Number(e.target.value)) ? Number(e.target.value) : null;
```

Reply:
good catch, thanks!! I'll update it